src/views/Main.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import BlogCard from '../components/BlogCard/BlogCard.js';
+
+
+import getBlog from '../services/blogs';
+
+export default function Main() {
+  const [blogs, setBlogs] = useState([]);
+
+  useEffect(()=>{
+    const fetchData = async () => {
+      const response = await getBlog();
+      setBlogs(response);
+    };
+    fetchData();
+  }, []); 
```

Comment:
Great job Brenden! It looks like you are getting super comfortable using hooks in your code, keep up the great work!